### DISCUSSION TITLE
Frontend modification

### DISCUSSION THREAD

**Discussion by miladganjali** [LiveHelperChat/livehelperchat#2162] [Q&A]
Hi @remdex,

Thank you for creating such an outstanding application! I'm a full-stack developer interested in contributing RTL support to your project. However, the codebase is quite extensive, spanning jQuery, Angular, React, and Svelte. Could you please guide me on how to modify, build, and publish my changes effectively?

I have set up the project locally using this [repository](https://github.com/LiveHelperChat/docker-standalone).

Thanks in advance for your guidance!

**Comment by remdex** [comment]
https://doc.livehelperchat.com/docs/development/quick-guide
